src/webpack/webpackTools.ts

```diff
@@ -147,10 +149,10 @@ export function createConfig(outPath: string, entryPoint: string | webpack.Entry
  * Will be removed once the RVM supports relative paths within app.json files
  */
 export const manifestPlugin = (() => {
-    const {NAME, PORT} = getProjectConfig();
+    const {NAME, PORT, IS_SERVICE} = getProjectConfig();
 
     return new CopyWebpackPlugin([{
-        from: './res/provider/app.json',
+        from: IS_SERVICE ? './res/provider/app.json' : './res/app.json',
```

Comment:
Why not just have a config option something like `MANIFEST_LOCATION` that is the path to the `app.json` that way we aren't hardcoding the location?

Reply:
An element of lazyness - but also, the manifest location should be somewhat consistent, it should always be in one of those two places.

Could use a util function for this though, rather than doing this inline? Not sure how many places we reference the provider location within the tooling.